ansible_base/lib/redis/client.py

```diff
@@ -63,6 +64,12 @@ def connect(self, index: int = 0) -> Union[Redis, RedisCluster]:
         except (IndexError, ValueError):
             pass
 
+        for file_setting in ['ssl_certfile', 'ssl_keyfile', 'ssl_ca_certs']:
+            file = kwargs.get(file_setting, None)
```

Comment:
Good to know! I feel like one of the functions like this will raise an exception if the value is not in the collection so I always just explicitly returned None.